### PR TITLE
Gophercloud update blog post

### PR DESCRIPTION
**Pull request by jrperritt**

### CODE REVIEW

File: src/site_source/_posts/2015-04-08-gophercloud-update.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- [Cloud Orchestration](https://github.com/rackspace/gophercloud/tree/master/rackspace/orchestration/v1)
+- [Cloud Networks](https://github.com/rackspace/gophercloud/tree/master/rackspace/networking/v2)
+- [RackConnect](https://github.com/rackspace/gophercloud/tree/master/rackspace/rackconnect/v3)
+
+A Getting-Started guide for Cloud Load Balancers [exists](https://developer.rackspace.com/docs/cloud-load-balancers/getting-started/), and similar guides for the other services are being created.
+
+<!-- more -->
+
+## Automatic Re-authentication
+Gophercloud now supports automatic re-authentication. This will enable long-running jobs to continue as normal if the users token expires during code execution. To enable it, simply set the `gophercloud.AuthOptions` field `AllowReauth` to `true`:
```
Comment (smashwilson) [inline src/site_source/_posts/2015-04-08-gophercloud-update.markdown]: "continue as normal" sounds a bit awkward to me; I'd say "continue normally" instead.

**Comment by smashwilson** [comment]
Looking good, just one tiny tweak from me (and it's just something that didn't scan right to me, feel free to disregard).

One thing that would be good to add would be a section on "how to get the new stuff" - both `go get -u` and the `godeps` route.

File: src/site_source/_posts/2015-04-08-gophercloud-update.markdown (hunk 1 above)
Comment (jrperritt) [inline src/site_source/_posts/2015-04-08-gophercloud-update.markdown]: :+1:

**Comment by jrperritt** [comment]
Good idea. I'll add a section for that.

File: src/site_source/_posts/2015-04-08-gophercloud-update.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```go
+opts := gophercloud.AuthOptions{
+  Username: "myUserName",
+  APIKey: "myAPIKey",
+  AllowReauth: true,
+}
+```
+
+## AllPages Method
+We've added a new method named `AllPages` that allows users to retrieve all the pages from a `List` call without iterating. For example, to retrieve all RackConnect Load Balancer Pools:
```
Comment (jamiehannaford) [inline src/site_source/_posts/2015-04-08-gophercloud-update.markdown]: might there be some confusion about the "without iterating" line? users still have to iterate. how about: "without worrying about pagination"?

**Comment by jamiehannaford** [comment]
LGTM :+1:

File: src/site_source/_posts/2015-04-08-gophercloud-update.markdown (hunk 2 above)
Comment (jrperritt) [inline src/site_source/_posts/2015-04-08-gophercloud-update.markdown]: :+1:

**Comment by smashwilson** [comment]
Cool. :shipit:

**Pull request merged**